core/src/main/java/com/microsoft/applicationinsights/channel/concrete/inprocess/InProcessTelemetryChannel.java

```diff
@@ -181,11 +210,42 @@ public void flush() {
         telemetryBuffer.flush();
     }
 
+    /**
+     * Sets the buffer size
+     * @param maxTelemetriesInBatch should be between MIN_NUMBER_OF_TELEMETRIES_PER_CONTAINER
+     *                              and MAX_NUMBER_OF_TELEMETRIES_PER_CONTAINER inclusive
+     */
+    public void setMaxTelemetriesInBatch(int maxTelemetriesInBatch) {
+        if (maxTelemetriesInBatch >= MIN_NUMBER_OF_TELEMETRIES_PER_CONTAINER &&
+                maxTelemetriesInBatch <= MAX_NUMBER_OF_TELEMETRIES_PER_CONTAINER) {
+            telemetryBuffer.setMaxTelemetriesInBatch(maxTelemetriesInBatch);
+        } else {
+            InternalLogger.INSTANCE.trace("Value %s is ignored, value should be between %s and %s, inclusive",
```

Comment:
It is not an error as it is not a critical issue and we can continue and on the other hand I don't want this message to get masked by other tracing messages.
So maybe we can just add warning to our internal logger?